.github/workflows/scripts/comments.mjs

```diff
@@ -75,6 +78,62 @@ export async function commentDiff (
   }
 }
 
+/**
+ * Generates comment for stats
+ *
+ * @param {GithubActionContext} githubActionContext
+ * @param {number} issueNumber
+ * @param {StatComment} statComment
+ */
+export async function commentStats (
+  githubActionContext,
+  issueNumber,
+  { titleText, markerText }
+) {
+  const reviewAppURL = getReviewAppUrl(issueNumber)
+
+  // File sizes
+  const fileSizeTitle = '### File sizes'
+  const fileSizes = await getFileSizes()
+  const fileSizeRows = Object.entries(fileSizes)
+    .map(([key, value]) => {
+      return [key, String(filesize(value.size, { base: 2 }))]
+    })
+  const fileSizeHeaders = ['File', 'Size']
+  const fileSizeTable = renderTable(fileSizeHeaders, fileSizeRows)
```

Comment:
We similarly loop `componentNamesWithJavaScript` in the review app to build a summary list

Should we save a job and build the rows for both review app and PR stats in `'govuk-frontend-stats'`?

https://github.com/alphagov/govuk-frontend/blob/5804103ad5d417222f3b114c8c2db4e694ba2801/packages/govuk-frontend-review/src/views/index.njk#L116-L117

Saves maintaining two sets of table building logic

Reply:
I'd like to get this one merged as is, preferably. I'll add an issue to do some optimised table building.